src/linkgraph/linkgraphschedule.cpp

```diff
@@ -50,14 +51,24 @@ void LinkGraphSchedule::SpawnNext()
 	}
 }
 
+/**
+ * Join the next finished job, if available.
```

Comment:
Copy/paste comment fail? (I.e. it is the same as the function below).

Reply:
Thanks, should be fixed now.